File: unit1/cli_lab/cli2.c
#include <cs50.h>
#include <ctype.h>
#include <stdio.h>
#include <stdlib.h>

int main(int argc, string argv[])
{
    if (argc != 2)
    {
        printf("Usage: ./cli2 <temp>\n");
        return 1;
    }

    float c = (float) strtod(argv[1], NULL);

    printf("F: %.1f\n", c * 9 / 5 + 32);
}

File: unit3/cli2.py
import sys

def main():
    if len(sys.argv) != 2:
        print("Usage: python cli2.py <temp>")
        sys.exit(1)

    c = float(sys.argv[1])

    print(f"F: {c * 9 / 5 + 32: .1f}")

main()